File: codigos_de_c#/burbuja.cs
using System;

class Program
{
    static void Main()
    {
        Random rand = new Random();
        int num = 10;
        int[] array = new int[num];

        // Llenado aleatorio
        for (int i = 0; i < num; i++)
        {
            array[i] = rand.Next(1, 101); // entre 1 y 100
        }

        Console.Write("Los numeros del arreglo son: ");
        foreach (int val in array)
        {
            Console.Write(val + " ");
        }

        // Ordenamiento burbuja (mayor a menor)
        for (int i = 0; i < num - 1; i++)
        {
            for (int j = 0; j < num - 1 - i; j++)
            {
                if (array[j] < array[j + 1])
                {
                    int temp = array[j];
                    array[j] = array[j + 1];
                    array[j + 1] = temp;
                }
            }
        }

        Console.WriteLine("\nLos numeros de mayor a menor: ");
        foreach (int val in array)
        {
            Console.Write(val + " ");
        }
    }
}

File: codigos_de_python/burbuja.py
import random

def main():
    num = 10
    array = []

    # Llenado aleatorio
    for i in range(num):
        array.append(random.randint(1, 100))  # 1 a 100

    print("Los numeros del arreglo son:", *array)

    # Ordenamiento burbuja (mayor a menor)
    for i in range(num - 1):
        for j in range(num - 1 - i):
            if array[j] < array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]

    print("Los numeros de mayor a menor:", *array)

main()
